src/sentry/static/sentry/app/components/forms/inputField.tsx

```diff
@@ -4,7 +4,7 @@ import React from 'react';
 import FormField from 'app/components/forms/formField';
 
 type InputFieldProps = FormField['props'] & {
-  placeholder: string;
+  placeholder?: string;
```

Comment:
What's the reason for making this optional?

Reply:
Sometimes only the field title (Label) is enough